tasks/test.py

```diff
@@ -76,13 +76,18 @@ def test(ctx, targets=None, coverage=False, build_include=None, build_exclude=No
     print("--- go generating:")
     generate(ctx)
 
+
+    if skip_licenses:
+        print("--- [skipping licenses]")
```

Comment:
When/Why would we want to skip the license check ? It seems like a problem in the making.

Reply:
I believe this was to workaround a problem that was addressed in the master branch. We can probably remove this now.

Reply:
@hush-hush It actually might be useful for local tests, we already have `skip_linters`, thoughts?

Reply:
Linters take a long time to run (especially on windows). The license linting is fast and it's the kind of things we really don't want to mess with. I'd say we should keep it.